src/data/backup.ts

```diff
@@ -319,6 +320,24 @@ export const computeBackupAgentName = (
 export const computeBackupSize = (backup: BackupContent) =>
   Math.max(...Object.values(backup.agents).map((agent) => agent.size));
 
+export type BackupType = "automatic" | "manual" | "addon_update";
+
+export const BACKUP_TYPE_ORDER: BackupType[] = [
+  "automatic",
+  "manual",
+  "addon_update",
+];
+
+export const computeBackupType = (backup: BackupContent): BackupType => {
+  if (backup.with_automatic_settings) {
+    return "automatic";
+  }
+  if (backup.addon_update) {
```

Comment:
From the core PR, it seems this is in a new metadata field?

Reply:
This is how core determines that a backup is for an add-on update:
https://github.com/home-assistant/core/blob/09cea6ce967da3deb26c159d0ba491f621dbf378/homeassistant/components/hassio/backup.py#L673

Reply:
So that would be?

```ts
if ("supervisor.addon_update" in backup.extra_metadata)
```

Reply:
Yes. :+1:

Reply:
The backup wasn't done when I made the PR. I will do the change according to the backend PR